meshroom/core/desc.py

```diff
@@ -587,6 +587,13 @@ def stopProcess(self, chunk):
     def processChunk(self, chunk):
         raise NotImplementedError('No processChunk implementation on node: "{}"'.format(chunk.node.name))
 
+class InputNode(Node):
```

Comment:
Not entirely satisfied with using "InputNode" for non-computable nodes because I feel like there could be other use cases, but I unfortunately don't have a much better idea to designate them 😒

Reply:
What do you think of "ValueNode" ?
I feel like that it might better because it would not be necessarily in input.